### DISCUSSION TITLE
unwanted "/" between the base url and the request path.

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#148] [Q&A]
Dlite places an unwanted "/" between the base url and the request path.  

![image](https://user-images.githubusercontent.com/22573455/223746160-dad17692-cec7-4870-b682-2e34d0a52842.png)

The API fails.
If I go to Chrome Developer Tools and remove that "/" than the API works fine.

How can I stop that "/"?

Folkert

**Comment by renaudpawlak** [comment]
``forecast?`` is part of the request, not the base URL.
- Base Url: ``https://api.open-meteo.com/v1``
- Request path: ``forecast?latitude=``
 
Or also, if you consider that forecast is part of the base url:
- Base Url: ``https://api.open-meteo.com/v1/forecast``
- Request path: ``?latitude=``
